### PR TITLE
Upgrades + Possible Bug Fixes. 

### PR DESCRIPTION
**Pull request by Phyco-Ninja**
No bully for Noob work plox. 🤕 
Anyways added extract user and reason in Message class (which I'm using on My LazyBot currently,, works fine tho,) ,, some possible ``` Chat Send Media Forbidden ``` Fixes in Some plugins, added support for new method in admin folders.

### CODE REVIEW

File: userge/core/types/bound/message.py [hunk 1]
```diff
@@ -114,6 +118,42 @@ def process_is_canceled(self) -> bool:
...
+    def extract_user_and_reason(self):
+        """ Extracts User and Reason
+        [NOTE]: This method checks for reply first.
+        On Success:
+            user (```str | int | None```) and reason (```str | None```)
+        """
+        user_e = ''
+        reason = ''
+        if self.reply_to_message:
+            user_e = self.reply_to_message.from_user.id
```
Comment (rking32) [inline userge/core/types/bound/message.py:131]: if `self.reply_to_message.from_user` is `None`
can you handle it too

File: userge/plugins/admin/gadmin.py [hunk 2]
```diff
@@ -139,21 +126,13 @@ async def ban_usr(message: Message):
...
     await message.edit("`Trying to Ban User.. Hang on!! ⏳`")
-    if message.reply_to_message:
-        user_id = message.reply_to_message.from_user.id
-        reason = message.input_str
+    user_id, reason = message.extract_user_and_reason
+    if not user_id:
+        await message.edit(
+            text="`no valid user_id or message specified,`"
+            "`do .help ban for more info` ⚠", del_in=5)
+        return
```
Comment (Krishna-Singhal) [inline userge/plugins/admin/gadmin.py:134]: hmm delete this else and give indents 😐

**Comment by Krishna-Singhal** [comment]
Huli fek. 😂😂

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

File: userge/core/types/bound/message.py (hunk 1 above)
Comment (Phyco-Ninja) [reply userge/core/types/bound/message.py:131]: U mean. 
```if self.reply_to_message and self.reply_to_message.from_user```?

File: userge/plugins/admin/gadmin.py [hunk 3]
```diff
@@ -139,21 +126,13 @@ async def ban_usr(message: Message):
...
     else:
```
Comment (Krishna-Singhal) [inline userge/plugins/admin/gadmin.py]: do delete this else and give indents. 😐

**Review by Krishna-Singhal — CHANGES_REQUESTED** [review]

**Pull request merged**